index.html

```diff
@@ -15,18 +15,28 @@
           <a class="logo" href="#">Abel Tsegaye</a>
         </logo>
         <img class="mobileMenu" src="images\IconMenu.png" alt="Menu" />
+        <ul class="desktopMenu">
+          <li class="desktopLink">Portfolio</li>
+          <li class="desktopLink">About</li>
+          <li class="desktopLink">Contact</li>
+          <li class="desktopLink">
+            <img class="mail" src="/images/Mail.svg" alt="Mail" />
+          </li>
+        </ul>
       </nav>
     </header>
 
     <!-- Headline -->
     <section class="headline">
```

Comment:
I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220547570-186243b5-4e1f-431b-bf71-d5f75c218271.png)

## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220547585-b93ff8a1-5706-4400-93a5-ea2b64683231.png)